scripts/install.sh

```diff
@@ -170,6 +181,14 @@ Supported env variables:
 EOF
 }
 
+function reporter {
+    if ! $DISABLE_INSTALLATION_TELEMETRY; then
+        echo "SUMOLOGIC_INSTALLATION_TOKEN=${SUMOLOGIC_INSTALLATION_TOKEN}" >> $INSTALLATION_LOGFILE
```

Comment:
Could we use some hash to not store token directly inside the file and send it as source header? Right now we are going to get one line with the token and then multiple lines which will be hard to say which installation token are related

https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/upload-logs/#supported-http-headers

I think `X-Sumo-Name` would be the best choice :thinking:

Reply:
Not 100% sure if `X-Sumo-Name` is the best choice; what about using `X-Sumo-Fields` instead with a value of `installation-token-sha256=<value>`?

Reply:
Whether I use `X-Sumo-Name` or `X-Sumo-Fields` for the headers I send to the endpoint I was provided with, that data doesn't seem to make it into Sumo.

I had to remove compression already because it turned out the `Content-Type` header is not supported by the endpoint. I wonder if whatever they put in place just ignores headers. I'll get in touch with the team in charge of that endpoint.

Reply:
The team has confirmed that those headers are not supported; it's still unclear to me whether or not they'll work on supporting them.

Can you see a different way to accomplish the same idea?

Reply:
If the endpoint treats all the data like multiline log I think it's fine, Otherwise I would put hash on the beginning of every line, so we do not loose the context

Reply:
The hosted collector (`otel-installation-logs-collector`) is setup with `multilineProcessingEnabled: true`, so I think it's fine indeed.